pages/docs/environments/gitpod.mdx

```diff
@@ -10,10 +10,69 @@ Gitpod is an open-source application that provides a prebuilt development enviro
 
 You can use Gitpod in the cloud. Please note, that Gitpod requires additional configuration to work on the authenticated pages in LinkFree. See instructions for [how to set up your OAuth app](/docs/environments/environment-variables) for more information.
 
-There are 2 ways you can open a repository with Gitpod:
+There are **2** ways you can open a repository with Gitpod.
+
+####1st Method: Prefix the url with `gitpod.io/#`
+
+- Go to the GitHub page of your forked LinkFree repository
+- Copy the url from the address bar and paste the code `gitpod.io/#` in front of it
```

Comment:
There's another way to do this that may be less steps.  You can simply replace the url with your github account. For example:
https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/

for me becomes...
https://gitpod.io/#https://github.com/amandamartin-dev/LinkFree/

wdyt?

Reply:
Just to see if I understood correctly, do you mean to write something like that:

`You can simply replace the url with your GitHub account. (e.g. https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/)` [or another significant url]

instead of these two lines:

```
- Go to the GitHub page of your forked LinkFree repository
- Copy the url from the address bar and paste the code `gitpod.io/#` in front of it
```

Am I correct?

Reply:
Potentially, I would test the flow on your end and if you think it would be a valid pathway to offer you could either add it in addition as another option or replace if you think it's a better solution.  Ultimately, whatever will get folks up and running!

Reply:
I think that must be well specified to the user that he needs to put the code snippet (`https://gitpod.io/#`) 
in front of the url of the forked repo (the one he wants to work on Gitpod) and not in front of the url of his GitHub profile/account. From my point of view, this info is essential and must be clearly specified to avoid misunderstandings and errors.

Clearly, I leave the final decision to you 😊